### DISCUSSION TITLE
Transfer of USDC from Solana to Ethereum becomes Wormhole spl USDC token with no liquidity. How to bridge it back?

### DISCUSSION THREAD

**Discussion by Tohta78** [wormhole-foundation/wormhole#453] [Developer Q&A] [answered]
Solana txn: 3uhTG537tzBPSDG3ZxPoTTRr7UbGkjSj1rugS9nXVWJeuUTaVRPwMbf33XABimfdPSzQuUy6DyzGPbcWR1x6FQE1
Ethereum txn: 0xfd949ad89e9c9171c31004ea66b896d617c3fcc0583b55e42696276d654437a5
The transfer of USDC from Sol to Eth can be found as Wormhole spl USDC token in my metamask. There is no liquidity which is useless as i cannot swap to ERC USDC. How do i transfer it back from Ethereum to Solana network? Wormhole bridge cannot find the spl USDC token to transfer.

**Comment by Sandybeach25** [comment]
Tohta78, i have the same problem, but with USDT.
Only i can not make Wormhole spl USDT visable in my Metamask.
It is there because i can seee it with Etherscan in my address, but there it is listed as (wormhole) so between brackets.
How did you make it visable in Metamask?

**Comment by Tohta78** [comment, marked as answer]
Hi,
 You can click on the wormhole and get the contract address to add into metamask.

The same address can also be manually added in wormhole bridge if you want to transfer it back to solana.

**Comment by leoluk** [comment]
Is this for v1 or v2 Wormhole?
